chess/xboard.py

```diff
@@ -296,11 +296,10 @@ def __init__(self, Executor=concurrent.futures.ThreadPoolExecutor):
         self.search_started = threading.Event()
 
         self.board = chess.Board()
-        self.chess960 = None
-        self.variant = None
 
         self.name = None
         self.author = None
+        self.supported_variants = ""
```

Comment:
Maybe this should start as an empty list?

Reply:
Ahh, make sense. Added.